### ISSUE TITLE
should not require AWS_ACCESS_KEY_ID

### ISSUE THREAD

**Issue by mogadanez** [mozart-analytics/sqsd#8]
in  AWS setup prefer to use IAM roles instead key/secret pairs

**Comment by mortizbey** [comment]
You are right, I will evaluate the code to see if we can use the normal AWS authentication chain instead of forcing to use AWS keys.

**Comment by mogadanez** [comment]
Just rewrite this repo to node.js https://www.npmjs.com/package/sqsd with this changes.
for me it takes much less memory to run

**Comment by mortizbey** [comment]
That's great! However we would appreciate some credit and backreference to this repository so that people that do not want to use node.js can have a JAVA based alternative.
